CIP-????/README.md

```diff
@@ -26,7 +26,7 @@ These changes will provide very substantial speed and size improvements to scrip
 The first designs of Plutus Core had native support for datatypes.
 In the interest of keeping the language as small and simple as possible, this was removed in favour of encoding datatype values using [Scott encoding][].
 
-Experiments at the time showed that the performance penalty of using Scott encoding over native datatypes was not too great.
+[Experiments](https://github.com/input-output-hk/plutus/blob/96fd25649107658fc911c53e347032bedce624e9/doc/notes/fomega/scott-encoding-benchmarks/results/test-results.md) at the time showed that the performance penalty of using Scott encoding over native datatypes was not too great.
```

Comment:
Is this sentence saying that "the performance penalty was smaller than expected / small enough to be practical" or "the performance penalty was disappointing"?

I originally read it as the former, but then got confused on second read.

Reply:
The former, I'll clarify.